src/pqueue.js

```diff
@@ -0,0 +1,102 @@
+// 'use strict'
+//
+// class Node {
+//   constructor ( value, priority ){
+//     this.arr = []
+//     this.value = value
+//     this.priority = priority
+//   }
+// }
+
+export default class PriorityQueue {
+  constructor(value) {
+    this.arr = value || []
```

Comment:
I try to use complete, intention revealing words as identifiers.  Consider `array` as a possible replacement for `arr`.  Having said that, I also try to avoid using identifiers that describe the structure the variable holds, rather then the purpose the variable serves.  Consider instead an identifier like `elements` or `collection`.